docs/theme/mlflow/static/css/mobile.css

```diff
@@ -0,0 +1,49 @@
+/*
+ * Custom CSSs for optimizing mobile view
+ *
+ * *** PLEASE DO NOT OVERUSE *** as having two different styles reduces maintainability.
+ */
+
+
+/* Mobile styling */
+@media only screen and (max-width: 768px) {
```

Comment:
overall i think it looks alright, i do wish there were a better solution but i also am not sure how much time it's worth to spend on it. it would be nice to be able to change the tab bar to be `display: flex` and remove the padding for mobile, but i'm not sure if this is really possible unless we use JS to add a class to it (which would involve even more custom code):

https://github.com/mlflow/mlflow/assets/148037680/9cfe203c-8eb3-454a-b714-6e34ebe8017d

Reply:
Makes sense, I think it's possible to tweak some tab styling, as we just copied tab.css rather than importing so we have some control. Will give it a try!

Reply:
@daniellok-db With some help from GPT, I think I made it? Not super clean way but probably it's ok.
<img width="437" alt="Screenshot 2023-12-05 at 9 24 33" src="https://github.com/mlflow/mlflow/assets/31463517/270e963c-f545-4626-b07a-b5590ad0cc51">

Reply:
I think this looks fine :) Let's ship it

Reply:
Awesome, could you override those non-doc-related tests and merge? I don't have permission to do so:)